legend-sdlc-generation-file-maven-plugin/src/main/java/org/finos/legend/sdlc/generation/file/FileGenerationMojo.java

```diff
@@ -286,6 +292,18 @@ private static ResolvedPackageableElementFilter resolvePackageableElementFilter(
         return new ResolvedPackageableElementFilter(resolvedElementsByPath, elementFilter.packages);
     }
 
+    private SDLC buildSDLCInfo()
+    {
+        MavenProject projectToInspect = mavenProject.getParent() != null ? mavenProject.getParent() : mavenProject;
+
+        AlloySDLC sdlcInfo = new AlloySDLC();
+        sdlcInfo.groupId = projectToInspect.getGroupId();
+        sdlcInfo.artifactId = projectToInspect.getArtifactId();
+        sdlcInfo.version = projectToInspect.getVersion();
```

Comment:
You should be able to get the version from the project directly without needing to go to the parent.

Reply:
Agreed, amended